straxen/plugins/bayes_peaklet_classification.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
+from scipy.special import logsumexp
+import strax
+import straxen
+
+class BayesPeakletClassification(strax.Plugin):
+    """
+    Bayes Peaklet classification
+    """
+    provides = 'peaklet_classification_bayes'
+    depends_on = ('peaklets',)
```

Comment:
This would imply retraining, and for most of the cases we focus on s1_prob

Reply:
Ah I was about to make the same suggestion as Daniel. There is one extra benefit and that would be that it would be much faster since we are running on much fewer Sx since the S2s are split over many peaklets by design

Reply:
Right, since we are doing classification we decided to switch to peak as you guys suggested